Rego/DefenderConfig.rego

```diff
@@ -617,42 +681,39 @@ tests[{
 SensitiveRulesNotBlocking[Rule.Name] {
     Rule := SensitiveRules[_]
     not Rule.BlockAccess
-    Policy := input.dlp_compliance_policies[_]
-    Rule.ParentPolicyName == Policy.Name
-    Policy.Mode == "Enable"
+    Rule.ParentPolicyName in SensitiveInfoPolicies
 }
 
-# Covers rules set to block, but inside policies set to
-# "TestWithNotifications" or "TestWithoutNotifications" that won't enforce the block
 SensitiveRulesNotBlocking[Rule.Name] {
     Rule := SensitiveRules[_]
-    Policy := input.dlp_compliance_policies[_]
-    Rule.ParentPolicyName == Policy.Name
     Rule.BlockAccess
-    startswith(Policy.Mode, "TestWith") == true
+    Rule.ParentPolicyName in SensitiveInfoPolicies
+    Rule.BlockAccessScope != "All"
 }
 
-SensitiveRulesNotBlocking[Rule.Name] {
-    Rule := SensitiveRules[_]
-    Rule.BlockAccess                              
-    Policy := input.dlp_compliance_policies[_]
-    Rule.ParentPolicyName == Policy.Name
-    count(error_rules) == 0
-    Rule.BlockAccessScope != "All"
+DefenderErrorMessage43(Rules) := GenerateArrayString(Rules, ErrorMessage) if {
```

Comment:
Nitpick: This is not the 43rd Defender Error message, this is the Error Message for Defender 4.3; Therefore, `DefenderErrorMessage4_3`